decidim-proposals/app/views/decidim/proposals/proposal_endorsements/update_buttons_and_counters.js.erb

```diff
@@ -1,4 +1,10 @@
 <% fully_endorsed= fully_endorsed?(proposal, current_user) %>
+<%# main page button %>
```

Comment:
Sorry to be such a PITA over this, but we're not using comments like this anywhere in the project. We could start a discussion whether that's needed or it actually reflects a design flaw (I'm more inclined to that last argument), but for the moment, let's comply with the current style please :)

Reply:
Or this means this part could be extracted in a partial/cell 😄

Reply:
I agree that it is actually a hint for a design flaw. I extracted the code into js methods.